erigon-lib/state/aggregator.go

```diff
@@ -295,6 +295,18 @@ func (a *Aggregator) OpenFolder() error {
 	a.dirtyFilesLock.Lock()
 	defer a.dirtyFilesLock.Unlock()
 	eg := &errgroup.Group{}
+	eg.Go(func() error {
+		dirtyDomainFiles, dErr := filesFromDir(a.dirs.SnapDomainDirty)
+		if dErr == nil {
+			for _, fn := range dirtyDomainFiles {
+				rmDirsErr := os.Remove(filepath.Join(a.dirs.SnapDomainDirty, fn))
```

Comment:
oh, no. taking my words back:
- we can't remove anything at startup - because RPCDaemon using same code with Erigon. And RPCDaemon must not remove files.
- And i don't want to introduce "readonly" mode of Agg 
- Let's remove files after merge (same as we doing now, for other garbage files)

Reply:
they are renamed during `integrateDirty` (not copied)
I personally dont like that approach at all - after restart need to rebuild files, cant merge commitment separately after eg accounts already merged, restart, just merge storage and the commitment instead of rebuilding and reindexing accounts..

Reply:
Agg still can scan sub-dir and use existing files as dirtyFiles. Just they will be ignored by “downloader torrent_create —all”.

or just return error if “downloader torrent_create —all” see overlap? What devops can do to finish merge? Stop and “erigon snapshots retire”?